### PR TITLE
Use default Docker syslog driver settings

### PR DESCRIPTION
**Pull request by yhahn**
**TL;DR** leaving out the explicit connection options for the `syslog` driver works on ECS. Hunches on why this wasn't working below...

------

@rclark I wasn't able to see any of the logging output of `ecs-conex` in the syslog of the container instance and had a hunch that `localhost` in the context of the container might be a bit suspect. I don't know for sure but these two clues made me think that connecting to the host instance this way wasn't going to fly.

- Docker docs on how the syslog driver behaves without options. The note about using a socket by default ... :thought_balloon: 

    > syslog-address specifies the remote syslog server address where the driver connects to. If not specified it defaults to the local unix socket of the running system.

- Various literature making it sound like establishing a connection back to the conatiner host isn't so straightforward...
  - https://github.com/docker/docker/issues/1143
  - http://stackoverflow.com/questions/24319662/from-inside-of-a-docker-container-how-do-i-connect-to-the-localhost-of-the-mach

### CODE REVIEW

**Comment by rclark** [comment]
:+1: This is a more generic setup. The prior state assumed that you were running a dedicated syslog listener that would handle logs for you in some fashion. 

I'll put a note in the readme about the log outputs to `/var/log/messages` on ECS-optimized AMIs and then merge here.

cc @jacquestardie
